ob_v3p0/impl/recommended-practices.md

```diff
@@ -0,0 +1,298 @@
+## Recommended Practices
+
+Here are a variety of recommended practices and considerations for the
+implementation of the Open Badges and CLR specifications. Conformance
+certification requirements ensure some important things, but a healthy ecosystem
+will develop out of the collaboration and experience of the implementor
+community. These practices will continue to be augmented as implementation
+proceeds. Implementers are encouraged to join the 1EdTech community and discuss
+how we can further improve our implementations and guidance together.
+
+### Selecting recipient and issuer identifiers, such as DID methods
```

Comment:
A similar section to this may be helpful for crypto algorithms?

Reply:
I'm not a cryptography expert. I sketched out such a section in commit [0b3f6e3](https://github.com/1EdTech/openbadges-specification/pull/501/commits/0b3f6e352f66a903630e1ccfc7e5155ffa8a3781) that can start the conversation.